tests/TestCase/Console/HelpFormatterTest.php

```diff
@@ -183,7 +183,7 @@ public function testHelpWithOptions(): void
         $parser->addOption('test', ['help' => 'A test option.'])
                ->addOption('number', [
                    'help' => 'The number',
-                   'default' => 2,
+                   'default' => '2',
```

Comment:
Option defaults must be boolean or a string.

Reply:
Will non-bool|string values cause errors? We might want to add this to the list of things we fix with rector, as I'm certain this will impact applications.

Reply:
It will throw a TypeError when creating ConsoleInputOption instance.

https://github.com/cakephp/cakephp/blob/5.0/src/Console/ConsoleOptionParser.php#L431

That class casts everything to string as well:

https://github.com/cakephp/cakephp/blob/5.0/src/Console/ConsoleInputOption.php#L120

Reply:
We could also uptype it and let it cast everything. However the getOption should probably change.